### ISSUE TITLE
Refactor issues_responder.py to be more funtional, and less procedural

### ISSUE THREAD

**Issue by TheSeanLavery** [TheSeanLavery/code-review-gpt#20]
Break things up into methods

**Comment by CodeReviewGPT** [comment]
I completely agree that breaking things up into methods is a great way to organize and simplify code. Not only does it make the code easier to read and understand, but it also makes it easier to maintain and debug. It's also a great way to keep related pieces of code together. Breaking up code into methods also helps to reduce complexity and can make it easier to optimize for performance.
